components/network/ntp/Makefile

```diff
@@ -26,12 +26,14 @@
 # Copyright (c) 2023, Niklas Poslovski
 #
 
+# for the moment the SHAKE128 digest fails
+USE_DEFAULT_TEST_TRANSFORMS= yes
```

Comment:
Is the `results-all.master` reproducible with this?

Reply:
yes